travis_ci/test_defaults.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+#
+# This file is part of the KubeVirt project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Copyright 2020 Red Hat, Inc.
+#
+set -ex
+
+templates_dir='dist/templates'
+default_label='template.kubevirt.io/default-os-variant=true'
+
+make generate
+oc create -f $templates_dir
+
+for os in `(cd $templates_dir && ls -1 *.yaml) | cut -d '-' -f 1 | sort -u`; do
```

Comment:
It is not enough to only iterate over the os portion of the template name, because every template covers multiple operating systems, for example, `rhel8` templates have these labels:
```
os.template.kubevirt.io/rhel8.0: "true"
os.template.kubevirt.io/rhel8.1: "true"
os.template.kubevirt.io/rhel8.2: "true"
os.template.kubevirt.io/rhel8.3: "true"
```
So the loop should iterate over all `os.template.kubevirt.io/*` within each template

Reply:
Got it, new version pushed!